src/fastoad/io/configuration/configuration.py

```diff
@@ -66,6 +67,8 @@ class for configuring an OpenMDAO problem from a configuration file
 
     def __init__(self, conf_file_path: Union[str, PathLike] = None):
         self._serializer: _IDictSerializer = _YAMLSerializer()
+        # Store imported classes
```

Comment:
This kind of comment suggests an action.
(and an empty line would be good for sake of clarity and consistency)
```suggestion

        # for storing imported classes
```

Reply:
Corrected in [b5e6c07](https://github.com/fast-aircraft-design/FAST-OAD/pull/559/commits/b5e6c07c475e0b4c6a90b747cb483b50b17db6f5)